src/fu-tool.c

```diff
@@ -1610,6 +1603,48 @@ fu_util_update(FuUtilPrivate *priv, gchar **values, GError **error)
 		fu_util_display_current_message(priv);
 	}
 
+	/* show warnings */
+	if (devices_latest->len > 0) {
+		fu_console_print_literal(priv->console,
+					 /* TRANSLATORS: message letting the user know no device
+					  * upgrade available */
+					 _("Devices with the latest available firmware version:"));
+		for (guint i = 0; i < devices_latest->len; i++) {
+			FwupdDevice *dev = g_ptr_array_index(devices_latest, i);
+			fu_console_print(priv->console, " • %s", fwupd_device_get_name(dev));
+		}
+	}
+	if (devices_unsupported->len > 0) {
+		fu_console_print_literal(priv->console,
+					 /* TRANSLATORS: message letting the user know no
+					  * device upgrade available due to missing on LVFS */
+					 _("Devices with no available firmware updates: "));
+		for (guint i = 0; i < devices_unsupported->len; i++) {
+			FwupdDevice *dev = g_ptr_array_index(devices_unsupported, i);
+			fu_console_print(priv->console, " • %s", fwupd_device_get_name(dev));
+		}
+	}
+	if (devices_pending->len > 0) {
+		fu_console_print_literal(priv->console,
+					 /* TRANSLATORS: message letting the user there is an update
+					  * waiting, but there is a reason it cannot be deployed */
+					 _("Devices with pending firmware updates: "));
```

Comment:
Pending I feel like is the wrong adjective here.

Shouldn't it be more explicit?
Something like "devices with firmware updates available that can't be installed"?

Reply:
That's a bit wordy. How about `Devices with updates that need user action:` or somethign like that

Reply:
Done!